core/src/components/action-sheet/action-sheet-interface.ts

```diff
@@ -26,4 +26,5 @@ export interface ActionSheetButton<T = any> {
   htmlAttributes?: { [key: string]: any };
   handler?: () => boolean | void | Promise<boolean | void>;
   data?: T;
+  disabled?: boolean;
```

Comment:
~ We should add a JSDoc comment discussing this property. Specifically to use it for buttons that are not the cancel action. This information should come through wherever the TS language server can parse the types, avoiding developers solely having to rely on the documentation on the site.

Reply:
Added in https://github.com/ionic-team/ionic-framework/pull/28723/commits/cff8ce7d413265b00b086d7e78bb4714db62b557